FocusEntity-Example/FocusEntity-Example/FocusARView.swift

```diff
@@ -17,8 +17,18 @@ class FocusARView: ARView {
   required init(frame frameRect: CGRect) {
     super.init(frame: frameRect)
     self.setupConfig()
-    self.focusEntity = FocusEntity(on: self, focus: .classic)
-//    self.focusEntity = FocusEntity(on: self, style: .colored(onColor: .red, offColor: .blue, nonTrackingColor: .orange))
+    //self.focusEntity = FocusEntity(on: self, focus: .classic)
```

Comment:
Let's leave the example project showing the classic version, with the texture version commented